.github/workflows/actuated-aarch64-test.yaml

```diff
@@ -0,0 +1,60 @@
+name: Actuated aarch64 test
+
+on: [push, pull_request]
+
+# Cancel any preceding run on the pull request.
+concurrency:
+  group: stream-test-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/criu-dev' }}
+
+jobs:
+  build:
+    runs-on: actuated-arm64-4cpu-3gb
+    strategy:
+      matrix:
+        target: [GCC=1, CLANG=1]
+
+    steps:
+    # https://gist.github.com/alexellis/1f33e581c75e11e161fe613c46180771#file-metering-gha-md
+    # vmmeter start
+    - name: Prepare arkade
+      uses: alexellis/arkade-get@master
+      with:
+        crane: latest
+        print-summary: false
+
+    - name: Install vmmeter
+      run: |
+        crane export --platform linux/arm64 ghcr.io/openfaasltd/vmmeter:latest | sudo tar -xvf - -C /usr/local/bin
+
+    - name: Run vmmeter
+      uses: self-actuated/vmmeter-action@master
+    # vmmeter end
+
+    - uses: actions/checkout@v4
+    - name: Run Tests ${{ matrix.target }}
+      # Following tests are failing on the actuated VMs:
+      # ################### 3 TEST(S) FAILED (TOTAL 463/SKIPPED 54) ####################
+      # * zdtm/static/change_mnt_context(ns)
+      # * zdtm/static/macvlan(ns)
+      # * zdtm/static/sched_policy00(ns)
+      ###################################### FAIL #####################################
+      #
+      #    ./change_mnt_context --pidfile=change_mnt_context.pid --outfile=change_mnt_context.out
+      #    45: ERR: change_mnt_context.c:23: mount (errno = 22 (Invalid argument))
+      #
+      #    Test output: ================================
+      #    Device "zdtmmvlan0" does not exist.
+      #    # the macvlan module is not available on the actuated VMs
```

Comment:
Could we add a `macvlan.checkskip` file to skip this test when the macvlan module is not available?